### DISCUSSION TITLE
Four Keys Dashboard connector

### DISCUSSION THREAD

**Discussion by beaujolaisruben** [GoogleCloudPlatform/fourkeys#425] [Q&A]
Hello,
I've successfully deployed the Four Keys project.
Now I would like to create the report.
In the Four Keys [video](https://www.youtube.com/watch?v=2rzvIL29Nz0) from Four Keys documentation Dina is using a "Four Keys connector", but I can't find it on Looker.
I tried with BigQuery connector and of course I don't have the report.
Could help, please ?
Do I have to use an alternative connector in Looker.
Thank you very much.
Ruben.

**Comment by colinfrei** [comment]
I'm not sure about the history, but since the setup includes a grafana dashboard, the connector might not be necessary anymore.

For me, running this command outputs the URL of the dashboard: 
`gcloud run services list --project $PROJECT_ID | grep grafana-dashboard- | awk '{print $2}'`
